### ISSUE TITLE
Too long distance between text and equation

### ISSUE THREAD

**Issue by wilkus27** [Connexions/webview#1903]
On staging.cnx.org (https://staging.cnx.org/contents/kRfPjKij@5.1:grqn3Xh2@2/11-3-The-Expenditure-Output-or-Keynesian-Cross-Model) is equation with blank spaces above and under of it.

![distance](https://user-images.githubusercontent.com/40228854/45103168-8e132a00-b12f-11e8-8e2c-6b20963ed85a.JPG)

**Comment by PiotrKozlowski** [comment]
This looks like content issue. @PatBoj or @Adrianltd can you confirm?

**Comment by Adrianltd** [comment]
Yes, I confirm that this is most probably caused by the content.
@openstaxalina @helenemccarron
